psga/s_aureus/check_metadata.nf

```diff
@@ -0,0 +1,21 @@
+/*
+ * Check the metadata in the database
+ */
+process check_metadata {
+
+  tag "${metadata}"
+
+  input:
+    path metadata
+
+  output:
+    path metadata, emit: ch_metadata
+
+  shell:
+  '''
+  metadata=!{metadata}
+
+  metadata_checked="check_metadata.done"
+  touch ${metadata_checked}
+  '''
+}
```

Comment:
This is not needed.

If you need ch_metadata, 
`ch_metadata = organise_metadata_sample_files.out.ch_metadata`
does this for you already.